packages/@yoda/network/index.js

```diff
@@ -0,0 +1,141 @@
+'use strict'
+
+/**
+ * @module @yoda/network
+ * @description Provides classes to manage network functions on the device.
+ */
+
+var EventEmitter = require('events').EventEmitter
+var FloraComp = require('@yoda/flora/comp')
+
+class NetworkAgent extends EventEmitter {
+  constructor () {
+    super()
+
+    this._remoteCallTarget = 'net_manager'
+    this._remoteCallCommand = 'network.command'
+    this._remoteCallTimeout = 60 * 1000
+
+    this._flora = new FloraComp('@network')
+    this._flora.init()
+  }
+
+  subscribe () {
+    this._flora.subscribe('network.status', (caps, type) => {
```

Comment:
See https://github.com/yodaos-project/yodart/blob/master/packages/%40yoda/flora/comp.js#L24-L44, flora-comp provides simple way to subscribe if calling the `init()`:

```js
this._flora.handlers = {
  'network.status': (args) => {
    // handler
  }
}
```

Reply:
Function subscribe is optional and currently only used in the custodian.